core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -352,6 +352,7 @@ class VersionInfo private[spark](
 class ApplicationEnvironmentInfo private[spark] (
     val runtime: RuntimeInfo,
     val sparkProperties: Seq[(String, String)],
+    val hadoopProperties: Seq[(String, String)],
```

Comment:
Looks it breaks mima but the constructor is private. Let's exclude it in mima. See the Jenkins test messages.

Reply:
Thank you! I will try it.